env/work/srw_python/srwlib.py

```diff
@@ -4056,6 +4333,10 @@ def srwl_wfr_emit_prop_multi_e(_e_beam, _mag, _mesh, _sr_meth, _sr_rel_prec, _n_
         if(workStokes == None):
             workStokes = SRWLStokes(1, 'f', meshRes.eStart, meshRes.eFin, meshRes.ne, meshRes.xStart, meshRes.xFin, meshRes.nx, meshRes.yStart, meshRes.yFin, meshRes.ny, doMutual)
 
+        # Erase the contents of .log file:
+        total_num_of_particles = nRecv * _n_part_avg_proc
+        srwl_save_status(0, total_num_of_particles, cores=nProc, particles_per_iteration=_n_part_avg_proc)
```

Comment:
Variables for progress monitoring. Will be used in Sirepo for progress bar of partially-coherent calculations (see https://github.com/radiasoft/sirepo/issues/329).